lib/jekyll/commands/serve/servlet.rb

```diff
@@ -27,7 +27,7 @@ def search_file(req, res, basename)
 
         #
 
-        def do_GET(req, res)
+        def do_get(req, res)
```

Comment:
I believe this is a special method so you can add a comment above the method to disable whatever it violates via `# rubocop:disable <copname>` e.g. `# rubocop:disable Metrics/LineLength`.